exchanges/stream/websocket.go

```diff
@@ -488,15 +509,26 @@ func (w *Websocket) trafficMonitor() {
 				w.setTrafficMonitorRunning(false)
 				w.Wg.Done()
 				return
-			case <-w.TrafficAlert:
-				if !trafficTimer.Stop() {
-					select {
-					case <-trafficTimer.C:
-					default:
+			case t := <-w.TrafficAlert:
+				if w.AuthConn != nil && w.AuthConn.GetURL() == t {
```

Comment:
I would like to default and error on a switch case if there is no match just for future implementations.

Reply:
I redid it if I understood you correctly.